mealplanner-ui/src/pages/Meals/Meals.tsx

```diff
@@ -34,20 +36,29 @@ const mealsQuery = graphql`
     gqLocalState {
       selectedMealTags
     }
+    # fragment name from PersonFavoriteMeals
+    ...PersonFavoriteMeals_favorites @arguments(slug: $slug)
+    gqLocalState {
+      selectedFavoriteMeals
+    }
   }
 `;
 
 export const Meals = () => {
   const [searchMeal, setSearchMeal] = useState<string>("");
   const [searchType, setSearchType] = useState('name');
+  const slug = getCurrentPerson().personSlug;
```

Comment:
For admin and meal designer, they should be able to select the user and view their favorite meals. Only for normal user it is current person.